### PR TITLE
Caliper

### PR DESCRIPTION
**Pull request by charest**

### CODE REVIEW

**Comment by charest** [comment]
@junghans Did you move the docker files?

**Comment by junghans** [comment]
Huh? The error is:
```
CMake Error at cinch/cmake/ProjectLists.txt:783 (add_executable):
  Cannot find source file:
    /home/flecsi/flecsale/libsrc/utils/test/caliper.cc
  Tried extensions .c .C .c++ .cc .cpp .cxx .m .M .mm .h .hh .h++ .hm .hpp
  .hxx .in .txx
Call Stack (most recent call first):
  CMakeLists.txt:38 (include)
CMake Error: CMake can not determine linker language for target: utils
CMake Error: Cannot determine link language for target "utils".
```
and that is correct, there no such file in this pull request!

File: config/packages.cmake [hunk 1]
```diff
@@ -106,13 +106,34 @@ else()
...
+
+find_package(Caliper QUIET)
+
+if (Caliper_FOUND)
+  option(ENABLE_CALIPER "Enable Caliper Support" ON)
+else()
+  option(ENABLE_CALIPER "Enable Caliper Support" OFF)
+endif()
+
+if(ENABLE_CALIPER)
```
Comment (junghans) [inline config/packages.cmake]: That seems wrong, if I  configure with `-DENABLE_CALIPER=ON` and Caliper isn't found, cmake will barf on undefined variables. I would do:
```
option(ENABLE_CALIPER "Enable Caliper Support" ${Caliper_FOUND})
if(ENABLE_CALIPER and NOT Caliper_FOUND)
  message(FATAL_ERROR "Caliper requested, but not found")
endif()
```

Comment (charest) [inline config/packages.cmake]: Oops.  I was looking at the wrong thing.

**Pull request merged**